Source/Readers/ImageReader/ImageDataDeserializer.h

```diff
@@ -271,7 +271,7 @@ class ImageDataDeserializer : public DataDeserializerBase
         errno = 0;
         description.m_classId = strtoull(label.c_str(), &eptr, 10);
         if (label.c_str() == eptr || errno == ERANGE)
-            RuntimeError("Cannot parse label value on line %" PRIu64 ", second column, in file %s.", lineIndex, mapPath.c_str());
+            RuntimeError("Invalid map file format, must contain 2 or 3 tab-delimited columns, line %" PRIu64 " in file %s.", lineIndex, mapPath.c_str());
```

Comment:
Why '2 or 3', does it not allow n-dimensional regression (e.g. 4-dimensional for bounding box regression)? The square loss should be able to handle this.